lua/core/menu/select.lua

```diff
@@ -1,37 +1,65 @@
+tabutil = require "tabutil"
+
 local m = {
   pos = 0,
   list = {},
-  len = "scan" 
+  favorites = {},
+  selected_script = "",
```

Comment:
I'm fine with tossing this -- it just felt a little weird calling `m.list[i+m.pos-2]` all over the place.